SU2_CFD/src/numerics/NEMO/NEMO_sources.cpp

```diff
@@ -324,6 +343,44 @@ CNumerics::ResidualType<> CSource_NEMO::ComputeAxisymmetric(const CConfig *confi
   residual[nSpecies+2] = yinv*rhov*H*Volume;
   residual[nSpecies+3] = yinv*rhov*U_i[nSpecies+nDim+1]/rho*Volume;
 
+  if (viscous) {
+
+    if (!rans){ turb_ke_i = 0.0; }
+
+    su2double sumJhs_y  = 0.0;
+    su2double sumJeve_y = 0.0;
+    su2double Mass      = 0.0;
+
+    for (iSpecies=0; iSpecies<nSpecies; iSpecies++)
+      Mass += V_i[iSpecies]*Ms[iSpecies];
+        
+    su2double heat_capacity_cp_i   = V_i[RHOCVTR_INDEX]+Ru/Mass;
+    su2double total_viscosity_i    = Laminar_Viscosity_i + Eddy_Viscosity_i;
+    su2double total_conductivity_i = ktr + kve + heat_capacity_cp_i*Eddy_Viscosity_i/Prandtl_Turb;      
+    su2double u                    = V_i[VEL_INDEX];
+    su2double v                    = V_i[VEL_INDEX+1];
+    su2double qy_ve                = kve*GV[TVE_INDEX][1];
+
+    /*--- Enthalpy and vib-el energy transport due to y-direction diffusion---*/
+    for (iSpecies = 0; iSpecies < nHeavy; iSpecies++) { 
+      sumJhs_y += (rho*Ds[iSpecies]*GV[RHOS_INDEX+iSpecies][1] - V_i[RHOS_INDEX+iSpecies]*Vector[1]) * hs[iSpecies];
+      sumJeve_y += (rho*Ds[iSpecies]*GV[RHOS_INDEX+iSpecies][1] - V_i[RHOS_INDEX+iSpecies]*Vector[1]) * eve_i[iSpecies];
+    }
```

Comment:
you can just ident this 2 lines for aesthetics (the += part)

Reply:
I would not spend too much time with that kind of alignment, eventually we will run clang-format on the entire code... The only thing you can do to make code look good is to write small functions.